### DISCUSSION TITLE
How to use 'steps' context in github actions toolkit?

### DISCUSSION THREAD

**Discussion by sakthi-ganapathy** [community/community#39642] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi,
I am trying to write a javascript action which requires validating the number of successful steps in the github workflow jobs.
How do we get the STEPS_CONTEXT into javascript action.

I am able to pass STEPS_CONTEXT as an input to javascript action as toJson(steps). But this seems like a hack. We have github context available in @actions/gihub toolkit., Similarly, Do we have an actions toolkit to get the steps context, so that i can validate the step statuses.

Thank you.

**Comment by ossanna16** [comment]
Hi there @sakthi-ganapathy and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
